parsers/cxx/cxx_tag.c

```diff
@@ -121,12 +121,7 @@ CXXToken * cxxTagSetTypeField(
 {
 	CXX_DEBUG_ASSERT(tag && pTypeStart && pTypeEnd,"Non null parameters are expected");
 
-	const char * szTypeRef0;
-
-	// "typename" is debatable since it's not really
-	// allowed by C++ for unqualified types. However I haven't been able
-	// to come up with something better... so "typename" it is for now.
-	static const char * szTypename = "typename";
+	const char * szTypeRef0 = NULL;
```

Comment:
Currently I think this is not acceptable. I think the value should be chosen by the author of the parser.
(Personally "type" is better than "typename".)